docs/source/index.rst

```diff
@@ -58,10 +58,21 @@ pyXem requires python 3 and conda - we suggest using the python 3 version of `Mi
       $ conda create -n pyxem
       $ conda activate pyxem
 
-Download the `source code <https://github.com/pyxem/pyxem>`__ and put it in a directory on your computer. The following commands will then install everything you need if entered into the anaconda promt (or terminal) when located in the pyxem directory:::
+If you have no intention of developing the pyxem repo, and simply using it your simplest option is then to type:::
 
+      $ conda install pyxem
+
+if you wish to use pip, that's also avaliable:::
+
+      $ conda install pip
+      $ pip install pyxem
```

Comment:
If the user is using a conda environment, it is better (and possibly less confusing) to stick using conda. That being said, it is worth mentioning pip for other environments/distributions.